docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,774 @@
+        "definition_id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+        "id": "04a98be3-7fb0-4cf5-af9a-31579c8b0e7d",
+        "descriptor_map": [
+            {
+                "id": "eu.europa.ec.eudiw.pid.it.1:unique_id",
+                "path": "$.vp_token.verified_claims.claims._sd[0]",
+                "format": "vc+sd-jwt"
+            },
+            {
+                "id": "eu.europa.ec.eudiw.pid.it.1:given_name",
+                "path": "$.vp_token.verified_claims.claims._sd[1]",
+                "format": "vc+sd-jwt"
+            }
+        ]
+    }
+  }
+
+Where the following parameters are used: 
+
+.. list-table::
+  :widths: 25 50
+  :header-rows: 1
+
+  * - **Name**
+    - **Description**
+  * - **vp_token**
+    - JWS containing a single (or an array) of Verifiable Presentation(s) in the form of JWS.
+  * - **presentation_submission**
+    - JSON Object contains mappings between the requested Verifiable Credentials and where to find them within the returned VP Token.
+  * - **state**
+    - Unique identifier provided by the Verifier inside the Authorization Request
+
+
+Below is a non-normative example of the ``vp_token`` decoded content, represented in the form of JWS header and payload, separated by a period:
+
+.. code-block:: text
+
+   {
+     "alg": "ES256",
+     "typ": "JWT",
+     "kid": "e0bbf2f1-8c3a-4eab-a8ac-2e8f34db8a47"
+   }
+   .
+   {
+    "iss": "https://wallet-provider.example.org/instance/vbeXJksM45xphtANnCiG6mCyuU4jfGNzopGuKvogg9c",
+    "jti": "3978344f-8596-4c3a-a978-8fcaba3903c5",
+    "aud": "https://verifier.example.org/callback",
+    "iat": 1541493724,
+    "exp": 1573029723,
+    "nonce": "n-0S6_WzA2Mj",
+    "vp": "<SD-JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>"
+   }
+
+Relying Party Entity Configuration
+---------------------------------------------
+According to the `Trust Model`_ section, the Verifier is a Federation Entity and must expose a well-known endpoint containing its Entity Configuration. 
+
+Below a non-normative example of the request made by the Wallet Instance to the *openid-federation* well-known endpoint to obtain the Relying Party Entity Configuration:
+
+.. code-block:: http
+
+  GET /.well-known/openid-federation HTTP/1.1
+  HOST: verifier.example.org
+
+
+Below is a non-normative response example:
+
+.. code-block:: text
+
+    {
+        "alg": "RS256",
+        "kid": "2HnoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs",
+        "typ": "entity-statement+jwt"
+    }
+    .
+    {
+        "exp": 1649590602,
+        "iat": 1649417862,
+        "iss": "https://rp.example.it",
+        "sub": "https://rp.example.it",
+        "jwks": {
+            "keys": [
+                {
+                    "kty": "RSA",
+                    "n": "5s4qi …",
+                    "e": "AQAB",
+                    "kid": "2HnoFS3YnC9tjiCaivhWLVUJ3AxwGGz_98uRFaqMEEs"
+                }
+            ]
+        },
+        "metadata": {
+            "wallet_verifier": {
```

Comment:
I didn't agree about inflating `openid_relying_party` metadata since this is a legacy metadata designed for legacy infrastructures

since we don't have the quorum for this metadata, I'd purpose `wallet_relying_party` even because OpenID4VP is more OAuth2 focused than OpenID, even if the standardization body is openid.

in fact openid4vp defines `client_metadata` for that, so the final proposal would be: do we have to use `client_metadata` or we're ok with `wallet_relying_party`?

Reply:
to be developed here https://github.com/italia/eidas-it-wallet-docs/issues/34